README.md

```diff
@@ -1,4 +1,5 @@
 # squash-your-commits
 Testing out the new option to squash
 
-Hello World
\ No newline at end of file
+Hello World
+FooBar
```

Comment:
Do you think it would be easier to read if there was a space between Foo and Bar?